src/devtools/client/debugger/src/client/index.js

```diff
@@ -20,13 +20,10 @@ import { initialBreakpointsState } from "../reducers/breakpoints";
 
 export async function loadInitialState() {
   const pendingBreakpoints = await asyncStore.pendingBreakpoints;
-  const tabs = { tabs: await asyncStore.tabs };
-
   const breakpoints = initialBreakpointsState();
 
   return {
     pendingBreakpoints,
-    tabs,
     breakpoints,
```

Comment:
aren't these fields also unneeded if they are in the replaySession state?

Reply:
We don't store pendingBreakpoints/breakpoints in the replay session state (yet).